doc/manual/plugins_global/saveimage.rst

```diff
@@ -0,0 +1,14 @@
+.. _sec-plugins-global-saveimage:
+
+SaveImage (Save File)
+=====================
+
+This brings up :ref:`local-plugin-saveimage` local plugin from the global
+Plugins menu for convenience.
+
+When done this way, the plugin is locked to the currently active image channel.
+To switch to another channel, follow these steps:
```

Comment:
@ejeschke , perhaps this is not clear enough. But when user follows the steps below, the plugin will update the channel and list only images associated with that channel.